applications/FSIapplication/python_scripts/Conformant_OneSideMap.py

```diff
@@ -1,10 +1,13 @@
 from __future__ import print_function, absolute_import, division #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
-# importing the Kratos Library
+
 from KratosMultiphysics import *
-from KratosMultiphysics.FSIApplication import *
-# Check that KratosMultiphysics was imported in the main script
 CheckForPreviousImport()
 
+if (Kernel().IsImported("FSIApplication")):
+    from KratosMultiphysics.FSIApplication import *
```

Comment:
Why do you check if it is imported and if yes, import it again?

Reply:
Because we want to be sure that you have imported all applications in the MainScript. Then you need to import again to have your stuff in your current Python file. Btw, we are about to add a new method to the __init__.py, meaning that this will be removed next week.

Reply:
Ok thx

Reply:
approving for now, but as we discussed this is very ugly. @philbucher we are adding a function to cleanup this, so don't replicate this in other places

Reply:
Sure
Pls let me know when u are ready then I can start using it too